CHANGELOG.md

```diff
@@ -23,6 +23,13 @@
   expressions like `filter(where:).isEmpty` instead of `contains(where:)`.  
   [Marcelo Fabri](https://github.com/marcelofabri)
 
+* Add `contains_over_range_not_nil` opt-in rule to prefer using `contains` over comparison of `range(of:)` to `nil`.  
```

Comment:
Can we rename the rule to `contains_over_range_nil_comparison` or something like that?